run_integration_tests.sh

```diff
@@ -6,7 +6,7 @@ pushd $rootdir
 
 SKIP_TEST_CONTAINER=0
 UPDATE_FEATURE_FILES_ONLY=0
-TEST_BRANCH=master
+TEST_BRANCH=use_latest
```

Comment:
https://github.com/algorand/go-algorand/pull/4154 is merged - switching go-algorand branches.

```suggestion
TEST_BRANCH=feature/box-storage
```